test/cookbooks/selinux_policy_test/files/default/p4d.te

```diff
@@ -0,0 +1,46 @@
+policy_module(p4d, 1.0.0)
```

Comment:
I'm having issues with creating my own type and after giving up I found this.
:two_hearts: